src/main/java/org/opensearch/knn/memoryoptsearch/faiss/FaissHnswGraph.java

```diff
@@ -187,4 +192,9 @@ public int consume(int[] ints) {
             throw new RuntimeException(e);
         }
     }
+
+    @Override
+    public int neighborCount() {
+        return 0;
```

Comment:
Why is this 0 ?

Reply:
Ignore this file, i added to avoid compilation issue from lucene